src/Playwright/Helpers/Paths.cs

```diff
@@ -29,9 +29,9 @@
 
 namespace Microsoft.Playwright.Helpers
 {
-    internal static class Paths
+    public static class Paths
     {
-        internal static string GetExecutablePath()
+        public static string GetExecutablePath()
```

Comment:
hmmm I think that I would rather duplicate the code than adding this to the API surface. Let's wait for more feedback here.